pkgs/top-level/all-packages.nix

```diff
@@ -7994,8 +7994,10 @@ in
       useMacosReexportHack = true;
     };
   };
-  llvm-polly = llvmPackages_latest.llvm-polly;
-  clang-polly = llvmPackages_latest.clang.override { cc = llvmPackages_latest.clang-polly-unwrapped; };
+
+  # llvm and clang are build with polly by default
```

Comment:
```suggestion
  # llvm and clang are built with polly by default
```

Reply:
After looking how this is usually done, I changed the lines to use throw. This prevents `llvm-polly`/`clang-polly`, but also provides an error message to the user. I think this is the best solution.

Thanks for finding the typo :)